<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIhmAppRs232c</class>
 <widget class="QMainWindow" name="CIhmAppRs232c">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CIhmAppRs232c</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cbPorts">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>221</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choix port : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOuvrir">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbRs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>421</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Communication s√©rie</string>
    </property>
    <widget class="QTextEdit" name="teTexte">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>401</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbEnvoyer">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTexte">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
